presidio-anonymizer/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN pipenv sync
 COPY . /usr/bin/${NAME}/
 
 EXPOSE ${FLASK_RUN_PORT}
-CMD pipenv run flask run --host 0.0.0.0
\ No newline at end of file
+CMD pipenv run python app.py --host 0.0.0.0
```

Comment:
Problem was that app was set with name by self.app = Flask(__name__)
but started from the global context, not self :
server = Server()
server.app.run(port=port)
As long as it works, it doesn't matter